Performance.md

```diff
@@ -181,16 +186,30 @@ Tests can also be broken into their own project.
 ------------                ------------
 ```
 
-One commonly asked question is "how big should a project be?".
-This is a lot like asking "how big should a function be?" or "how big should a class be?" and, to a large extent, it comes down to experience.
-One familiar way of splitting up JS/TS code is with folders.
-As a heuristic, if things are related enough to go in the same folder, they belong in the same project.
-Beyond that, avoid massive or tiny projects.
-If one project is larger than all the others combined, that's a warning sign.
-Similarly, it's best to avoid having dozens of single-file projects because the overhead adds up.
-
 You can [read up more about project references here](https://www.typescriptlang.org/docs/handbook/project-references.html).
 
+## Existing Code
+
+When a workspace becomes so large that it's hard for the editor to handle (and you've used [performance tracing](#providing-performance-traces) to confirm that there are no hotspots, making scale the most likely culprit) , it can be helpful to break it down into a collection of projects that reference each other.
```

Comment:
```suggestion
When a workspace becomes so large that it's hard for the editor to handle (and you've used [performance tracing](#providing-performance-traces) to confirm that there are no hotspots, making scale the most likely culprit), it can be helpful to break it down into a collection of projects that reference each other.
```

Reply:
How would a user know that scale is the culprit?

Reply:
The lack of hotspots.  That's what I was attempting to convey with the parenthetical, so please feel free to reword it to make that clearer.

Reply:
I'm going to merge so I don't lose track of this.  Please feel free to reword in a future PR.